src/components/Talk/ParameterSlider.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+  <div>
+    <div class="parameter-header">
+      <label :for="`parameter-${sliderKey}`" class="parameter-label">
+        {{ label }}
+      </label>
+      <input
+        :id="`parameter-${sliderKey}`"
+        :value="inputValueFixed"
+        class="parameter-input"
+        type="text"
+        @change="onUpdate"
+        @blur="onUpdate"
+      />
+    </div>
+    <BaseSlider
+      v-model="inputValue"
+      :min
+      :max
+      :step
+      :scrollStep
+      @valueCommit="$emit('update:modelValue', $event)"
+    />
+  </div>
+</template>
+
+<script setup lang="ts">
+import { computed, ref, watchEffect } from "vue";
+import BaseSlider from "@/components/Base/BaseSlider.vue";
+
+const props = defineProps<{
+  sliderKey: string;
+  min: number;
+  max: number;
+  step: number;
+  scrollStep: number;
+  label: string;
+  modelValue: number;
+}>();
+
+const inputValue = ref(props.modelValue);
+watchEffect(() => {
+  inputValue.value = props.modelValue;
+});
+const inputValueFixed = computed(() => inputValue.value.toFixed(2));
+
+const emit = defineEmits<{
+  "update:modelValue": [value: number];
+}>();
+
+const onUpdate = (event: Event) => {
```

Comment:
このonPudate、input由来かslider由来かどっちを想定してるかわかりづらいかも。
あとこれは割と慣習なんですが、イベントをハンドリングする時は`handle[Event名]`とするのが一般的かもです。引数として親から与えられるもの（イベントを通知するもの）が`on～`というイメージ･･･を持ってます！

```suggestion
const handleInputUpdate = (event: Event) => {
```